src/test/java/com/salesforce/dynamodbv2/DocGeneratorRunner.java

```diff
@@ -214,78 +224,120 @@ void byTableBySharedTable() {
+            "bySharedTableByTable",
+            docsChainsDir + "/bySharedTableByTable",
+            mtContext,
+            () -> amazonDynamoDB,
+            isLocalDynamo,
+            false,
+            ImmutableMap.of("na", physicalAmazonDynamoDB)).runAll();
     }
 
     /*
      * table -> sharedtable -> logger -> account
      */
     @Test
     void byTableBySharedTableByAccount() {
-        if (skipAccountTest) return;
+        if (skipAccountTest) {
+            return;
+        }
         AmazonDynamoDB accountAmazonDynamoDB = getAccountBuilder();
         AmazonDynamoDB amazonDynamoDB =
-                getTableBuilder().withAmazonDynamoDB(
+            getTableBuilder().withAmazonDynamoDB(
                 getBySharedTableBuilder().withAmazonDynamoDB(
-                getLoggerBuilder().withAmazonDynamoDB(
-                accountAmazonDynamoDB).build()).build()).build();
+                    getLoggerBuilder().withAmazonDynamoDB(
+                        accountAmazonDynamoDB).build()).build()).build();
         new DocGenerator(
-                "byTableBySharedTableByAccount",
-                docsChainsDir + "/byTableBySharedTableByAccount",
-                mtContext,
-                () -> amazonDynamoDB,
-                isLocalDynamo,
-                false,
-                getAccounts()).runAll();
+            "byTableBySharedTableByAccount",
+            docsChainsDir + "/byTableBySharedTableByAccount",
+            mtContext,
+            () -> amazonDynamoDB,
+            isLocalDynamo,
+            false,
+            getAccounts()).runAll();
     }
 
     /*
      * sharedtable -> table -> logger -> account
      */
     @Test
     void bySharedTableByTableByAccount() {
-        if (skipAccountTest) return;
+        if (skipAccountTest) {
+            return;
+        }
         AmazonDynamoDB table =
-                getTableBuilder().withAmazonDynamoDB(
+            getTableBuilder().withAmazonDynamoDB(
                 getLoggerBuilder().withAmazonDynamoDB(
-                getAccountBuilder()).build()).build();
+                    getAccountBuilder()).build()).build();
         AmazonDynamoDB amazonDynamoDB =
-                getBySharedTableBuilder().withAmazonDynamoDB(table).build();
+            getBySharedTableBuilder().withAmazonDynamoDB(table).build();
         new DocGenerator(
-                "bySharedTableByTableByAccount",
-                docsChainsDir + "/bySharedTableByTableByAccount",
-                mtContext,
-                () -> amazonDynamoDB,
-                isLocalDynamo,
-                false,
-                getAccounts()).runAll();
+            "bySharedTableByTableByAccount",
+            docsChainsDir + "/bySharedTableByTableByAccount",
+            mtContext,
+            () -> amazonDynamoDB,
+            isLocalDynamo,
+            false,
+            getAccounts()).runAll();
+    }
+
+    private AmazonDynamoDB getAccountBuilder() {
```

Comment:
what happened here? method (and inner class) reordering?

Reply:
We are going to need to understand the google style standard with respect to method ordering.  I kind of like the idea of it.  On the other hand, I don't like what the PR looks like when you change the visibility of the method and have to move it.

Reply:
one option then is that we do all changes (including changing visibility changes) and ctrl-alt-L change *except* for reorderings in one change list, followed by the full ctrl-alt-L modification (i.e., reorderings due to visibility), for exactly the reason of readability in diffs in code reviews

Reply:
I'm less concerned about the way this particular set of commits is going to look since it will be well known that it was part of a one-time effort to confirm to our checkstyle.  I'm more concerned in general about the idea that, going forward, if I change the implementation and visibility of a method in one commit, then it's difficult for anyone to see what the implementation change was because of the effect of the move.